### ISSUE TITLE
Write/Test Example YAMLs Converting High-Resolution Radiosondes BUFR to IODA

### ISSUE THREAD

**Issue by PraveenKumar-NOAA** [JCSDA-internal/ioda-converters#1114]
## Description

**The purpose of this issue is to:**

- Write example YAMLs  that can convert the High-Resolution Radiosondes BUFR files, using mnemonics determined in #1113 for the following subsets:

1. NC002101 -- MSG TYPE 002-101 RAWINSONDE - FIXED LAND
2. NC002102 -- MSG TYPE 002-102 RAWINSONDE - MOBIL LAND
3. NC002103 -- A51103 | MSG TYPE 002-103 RAWINSONDE - SHIP
4. NC002104 -- A51104 | MSG TYPE 002-104 DROPWINSONDE
5. NC002105 -- A51105 | MSG TYPE 002-105 PIBAL

- Use these YAMLs to convert into an IODA-formatted netCDF file
- Add outputs to the ioda-converter testoutput directory
- Add tests to the CMakeLists.txt files
- Have the tests pass

## Requirements

- All important mnemonics need to be known. #1113 will need to be finished

## Acceptance Criteria (Definition of Done)

- It will be finished when all files have been checked and tests have passed.

## Dependencies

- The bufr2ioda.x.

**Comment by PraveenKumar-NOAA** [comment]
YAMLs were created for the high-resolution data, once validation is done of the obs from the output NetCDF files, a PR will be submitted.

**Comment by PraveenKumar-NOAA** [comment]
Done, a PR is merged into the develop.
